R/write_checklist.R

```diff
@@ -1,5 +1,18 @@
 #' Write a check list with allowed issues in the source code
 #'
+#' First run `x <- checklist::check_package()` or
+#' `x <- checklist::check_source()`.
+#' These commands runs the checks and store the CheckList object in the variable
+#' `x`.
+#' Next you can store the configuration with `checklist::write_checklist(x)`.
+#' This will first list any existing allowed warnings or notes.
+#' For every one of them, you choose whether you want to keep it or not.
+#' Next, the function presents every new warning or note which can you allow.
+#' If you choose to allow a warning or note, you must provide a motivation.
+#' Please provide a sensible motivation.
```

Comment:
Can you give an example? I don't understand what you are referring to.

Reply:
The example that comes to mind is the case where I used `write_checklist()` to allow an error that happened locally due to the check not finding the RStudio pandoc installation on Windows. Pushing that list of allowed errors to the repo gave trouble because the error did not happen during online check_package (where pandoc is installed and available). Not the best example I know, but I'm pretty sure we talked about similar issues (for which I don't recall the exact details).

Reply:
The correct solution for this example is to fix your installation. The problem is not OS related by rather machine related.

Reply:
I've add this as a caveat in https://github.com/inbo/checklist/pull/38/commits/43e78a914ddd90d2f7766079e23113caaedbbfd4